third_party/canvaskit/BUILD.gn

```diff
@@ -15,6 +15,9 @@ wasm_toolchain("canvaskit") {
     skia_use_libjpeg_turbo_decode = true
     skia_use_libpng_decode = true
     skia_use_libwebp_decode = true
+
+    # Disable LTO.
+    enable_lto = false
```

Comment:
If this is going to be the same for all CanvasKit builds, you can also add it to [`tools/gn`](https://github.com/flutter/engine/blob/main/tools/gn#L607). But this is good too.

Reply:
Adding it to `tools/gn` would conflict with skwasm, which must always be built with LTO enabled